controllers/doguDeleteManager_test.go

```diff
@@ -87,12 +82,12 @@ func TestNewDoguDeleteManager(t *testing.T) {
 		cesRegistry.On("DoguRegistry").Return(doguRegistry)
 
 		// when
-		doguManager, err := NewDoguDeleteManager(client, operatorConfig, cesRegistry)
+		doguManager, err := NewDoguDeleteManager(client, cesRegistry)
 
 		// then
 		require.NoError(t, err)
 		require.NotNil(t, doguManager)
-		mock.AssertExpectationsForObjects(t, cesRegistry, doguRegistry)
+		// mock.AssertExpectationsForObjects(t, cesRegistry, doguRegistry)
```

Comment:
I think this should not be commented out. If it is no longer necessary then it should be removed.